android/settings.gradle

```diff
@@ -1,7 +1,3 @@
-// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Why remove the copyright header?

Reply:
This file was autogenerated, so leaving as is